src/toil/test/sort/sortTest.py

```diff
@@ -21,6 +21,7 @@
 import subprocess
 
 # Python 3 compatibility imports
+import sys
```

Comment:
Nit: maybe move this up one block? I don't think this is a Python 3 compatibility import.